docs/migration_guide.md

```diff
@@ -87,7 +87,7 @@ spec:
 ```
 
 ### Transition guide for state feedback
-Below the example for state feedback ('techCode' is optional and available only in case of failure). For transition to JSON format all 3 fields from 'state' section are important (except 'techCode') and their corresponding places in JSON are 'status' and 'message' in secions 'payload' and 'actions', and 'progress' in section 'actions'. Detailed list of available payload status and action status can be found in [link](docs/bfb.md) because for vehicles orchestration more sub-states were introduces for fine-grained reporting of self-update progress.
+Below there is an example for state feedback ('techCode' is optional and available only in case of failure). For transition to JSON format all three fields from 'state' section are important (except 'techCode') and their corresponding places in JSON are 'status' and 'message' in sections 'payload' and 'actions', and 'progress' in section 'actions'. The detailed list of the available payload status and action status can be found in [link](docs/bfb.md) because for vehicle orchestration more sub-states were introduced for fine-grained reporting of self-update progress.
```

Comment:
I would say "... for fine-grained reporting the self-update progress." or perhaps better:
"for a fine-grained report of the self-update progress."  My English is not the best...

Reply:
Fixed :)